zeus/config.py

```diff
@@ -33,7 +31,6 @@
 mail = Mail()
 nplusone = NPlusOne()
 redis = Redis()
-sentry = Sentry(logging=True, level=logging.ERROR, wrap_wsgi=True)
```

Comment:
I am not sure if having the level here at `error` and further down at `warn` was intentional fine-tuning. It's now `error` for all loggers